lib/money/money.rb

```diff
@@ -301,7 +305,19 @@ def self.from_amount(amount, currency = default_currency, bank = default_bank)
   #   Money.new(100, "USD") #=> #<Money @fractional=100 @currency="USD">
   #   Money.new(100, "EUR") #=> #<Money @fractional=100 @currency="EUR">
   #
-  def initialize(obj, currency = Money.default_currency, bank = Money.default_bank)
+  def initialize(
+    obj, currency = Money.default_currency, bank = Money.default_bank,
+    infinite_precision = nil
```

Comment:
This formatting is awkward to read. Either put them all on one line, or one line per argument.
Another way to shorten the method declaration is to have arguments default to `nil`, and then set the default values within the `initialize` method if the passed in value is `nil`. That also has the benefit of preventing the internal value from every being `nil` (if that is desired behavior).

Reply:
well, to be totally backward compatible we could leave it as is, but fix the formatting, and change the behavior in a newer version.